data_steward/test/run_tests.sh

```diff
@@ -8,17 +8,13 @@ subset="all"
 
```

Comment:
@calbach circleci build is failing, not sure if these edits in this file is the reason? If not since these are all shell scripts (with no unit testing needed) I can go ahead and squash and merge. Just wanted to double check.

Reply:
Rerunning now, I think this broke on account of some local integration testing I was doing.